homeassistant/components/izone/climate.py

```diff
@@ -112,13 +118,15 @@ def init_controller(ctrl: Controller):
     )
 
 
-def _return_on_connection_error(ret=None):
-    def wrap(func):
-        def wrapped_f(*args, **kwargs):
-            if not args[0].available:
+def _return_on_connection_error(
+    ret: _T = None,  # type: ignore[assignment]
```

Comment:
`Mypy` complains about the implicit Optional. However as `_T` is a TypeVar, I believe this should be fine and is a false-positive.